soc/intel/intel_adsp/cavs/include/pm_data.h

```diff
@@ -0,0 +1,15 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/*
+ * Copyright (c) 2024 Intel Corporation
+ *
+ * Author: Tomasz Leman <tomasz.m.leman@intel.com>
+ */
+
+#ifndef __INTEL_PM_DATA_H__
+#define __INTEL_PM_DATA_H__
+
+/* Currently, this header file serves only as a placeholder. This is intended to avoid a situation
+ * where including this header would need to be conditional with an ifdef.
+ */
+
+#endif /* __INTEL_PM_DATA_H__ */
```

Comment:
no needed for this header.

Reply:
@ceolin this placeholder avoids an `#idef ... #include` in multiprocessing.c (or anywhere else). This was discussed in https://github.com/zephyrproject-rtos/zephyr/pull/66092#discussion_r1460105808 and others.

```C
#if CONFIG_ADSP_DYNAMIC_CLOCK_SWITCHING
#include <pm_data.h>
#endif
```

Reply:
sorry, I was not clear about it. I don't think the whole idea of pm_data is needed, so this header is shouldn't exist.